tals/arin.tal

```diff
@@ -4,7 +4,13 @@ This file isn’t ARIN’s TAL but rather a placeholder. Until you replace
 it with the actual TAL you can find at the above URL, Routinator will
 not run.
 
-Please download the TAL in RFC 7730 format and copy it here.
+Please download the TAL in RFC 7730 format after reviewing the usage agreement
+at https://www.arin.net/resources/manage/rpki/tal/ and copy it here.
+
+Example (assumes system-wide routinator installation):
```

Comment:
I’d like to suggest a slightly more explicit wording:

```
For a system that stores the TALs in /etc/routinator/tals, this can be
achieved by running:
```